runner/app/live/trickle/trickle_subscriber.py

```diff
@@ -4,9 +4,9 @@
 import sys
 
 class TrickleSubscriber:
-    def __init__(self, url: str, max_retries=5):
+    def __init__(self, url: str, start_seq=-2, max_retries=5):
```

Comment:
Is this feature already implemented on the O side? Does `-2` mean getting the last "complete" segment and not the current (next) one being pushed? Or will `-1` pick the next seq even if it hasn't started yet?

Reply:
nvm just found https://github.com/livepeer/go-livepeer/pull/3552/files